CONTRIBUTING.md

```diff
@@ -4,15 +4,16 @@ For bugs, feel free to open issues or contact us directly. Thank you for your su
 
 ## Pull Request guideline
 
-Even though we will gladly assist you in finishing up your PR, try to
+Even though we will gladly assist you in finishing up your PR, try to:
+
 - keep all the crates compiling with *stable* rust (hide the eventual non-stable code under [`cfg`s](https://github.com/AFLplusplus/LibAFL/blob/main/libafl/build.rs#L26))
 - run `cargo +nightly fmt` on your code before pushing
-- check the output of `cargo clippy --all` or `./clippy.sh`
+- check the output of `cargo clippy --all` or `./scripts/clippy.sh` (On windows use `./scripts/clippy.ps1`)
```

Comment:
.\scripts\clippy.ps1 ?

Reply:
I was under the impression that both work on Windows at least in Powershell but \ is probably better, yes